x/gov/endblocker_test.go

```diff
@@ -279,3 +279,44 @@ func TestProposalPassedEndblocker(t *testing.T) {
 	resTags := EndBlocker(ctx, input.keeper)
 	require.Equal(t, sdk.MakeTag(tags.ProposalResult, tags.ActionProposalPassed), resTags[1])
 }
+
+func TestEndBlockerProposalHandlerFailed(t *testing.T) {
```

Comment:
This test should have more comments within - briefly describing what each block is doing.